src/integrators/exponential_rk_integrators.jl

```diff
@@ -10,17 +10,17 @@
   integrator.k[2] = zero(integrator.fsalfirst)
 end
 
-@inline @muladd function perform_step!(integrator,cache::LawsonEulerConstantCache,f=integrator.f)
-  @unpack t,dt,uprev,u,k = integrator
+@inline function perform_step!(integrator,cache::LawsonEulerConstantCache,f=integrator.f)
+  @unpack t,dt,uprev,u = integrator
   rtmp = integrator.fsalfirst
   A = f[1]
-  u = expm(dt*A)*(@. uprev + dt*rtmp)
+  @muladd u = expm(dt*A)*(@. uprev + dt*rtmp)
```

Comment:
Oh yes, now I remember you made it handle that. Cool. Just making sure it would stay correct 👍